docs/developer/extending/webhooks/ip-addresses.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Saleor IP Addresses
+---
+
+## Allowlisting Saleor IP Addresses
+
+Depending on your security requirements, it may be necessary to include Saleor's IP addresses to your firewall's allowlist for receiving webhook events.
```

Comment:
Depending on your security requirements, including Saleor’s IP addresses to your firewall’s allowlist for receiving webhook events may be necessary.